streaming/java/streaming-state/src/main/java/org/ray/streaming/state/strategy/AbstractTransactionState.java

```diff
@@ -0,0 +1,55 @@
+package org.ray.streaming.state.strategy;
+
+import java.util.Map;
+import org.ray.streaming.state.ITransactionState;
+import org.ray.streaming.state.backend.StorageMode;
+import org.ray.streaming.state.store.IKVStore;
+
+
+/**
+ * This class support ITransactionState.
+ */
+
```

Comment:
Adding informative java doc at class level: 

Based on the given StorageMode, different implementation instance of the AbstractStateStrategy class will be created. All method calls will be delegated to the strategy instance.

Reply:
ok